examples/tutorials/02_pipeline_operations.ipynb

```diff
@@ -15,7 +15,6 @@
    "source": [
```

Comment:
<p><em>Some</em>how, please, reformulate <em>some </em>parts of the first sentence to avoid repeating<em> some</em>what annoying pattern.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>